helm/charts/kratos/templates/deployment.yaml

```diff
@@ -66,6 +66,7 @@ spec:
           args: [
             "serve",
             "all",
+            "--watch-courier", # Will be configurable, check out https://github.com/ory/k8s/issues/285
             {{- if .Values.kratos.development }}
```

Comment:
```suggestion
            "all",
            {{- if .Values.kratos.config.courier.watch }}
            "--watch-courier", # Will be configurable, check out https://github.com/ory/k8s/issues/285
            {{- end }}
            {{- if .Values.kratos.development }}
```

Reply:
Thanks for the reply @Demonsthere. 

However, I have decided to close this PR in favor of a proper one #289. 

It's because I realize:

1. the changes in this PR will make user even more confused
2. More work is needed to make this chart usable. 

so, if you don't mind, please check out #289. 😃